src/Arcus.Security.Providers.AzureKeyVault/Authentication/ManagedServiceIdentityAuthenticator.cs

```diff
@@ -8,12 +8,24 @@ namespace Arcus.Security.Providers.AzureKeyVault.Authentication
     /// <summary>
     ///     Azure Key Vault authentication by using Azure Managed Service Identity
     /// </summary>
-    public class ManagedServiceIdentityAuthenticator : IKeyVaultAuthenticator
+#pragma warning disable 618
+    public class ManagedServiceIdentityAuthenticator : IKeyVaultAuthentication, IKeyVaultAuthenticator
+#pragma warning restore 618
     {
         /// <summary>
         /// Authenticates with Azure Key Vault
         /// </summary>
         /// <returns>A <see cref="KeyVaultClient"/> client to use for interaction with the vault</returns>
+        public async Task<IKeyVaultClient> AuthenticateClient()
+        {
+            IKeyVaultClient client = await Authenticate();
+            return client;
+        }
+
+        /// <summary>
+        ///     Authenticates with Azure Key Vault
```

Comment:
Let's align formatting with current docs or update the current ones with this approach

Reply:
This goes for all others as well

Reply:
It's ok for now